### PR TITLE
Add support for 9.0.0 version format

### PR DESCRIPTION
**Pull request by jkutner**

### CODE REVIEW

File: bin/java [hunk 1]
```diff
@@ -136,8 +136,10 @@ _get_jdk_download_url() {
     local jdkUrl=$(eval echo \$JDK_URL_1_${jdkVersion})
   elif [ "$(expr "${jdkVersion}" : '^1.[6-9]')" != 0 ]; then
     local jdkUrl="${JDK_BASE_URL}/openjdk${jdkVersion}.tar.gz"
+  elif [ "${jdkVersion}" = "9+181" ] || [ "${jdkVersion}" = "9.0.0" ]; then
+    local jdkUrl="${JDK_BASE_URL}/openjdk9-181.tar.gz"
```
Comment (hunterloftis) [inline bin/java]: Is this intentionally changing "9+181" to "9-181"?

File: bin/java [hunk 2]
```diff
@@ -136,8 +136,10 @@ _get_jdk_download_url() {
...
   elif [ "$(expr "${jdkVersion}" : '^9')" != 0 ]; then
-    local jdkUrl="${JDK_BASE_URL}/openjdk$(echo "$jdkVersion" | sed -e 's/9+/9-/g').tar.gz"
+    local jdkUrl="${JDK_BASE_URL}/openjdk${jdkVersion}.tar.gz"
```
Comment (hunterloftis) [inline bin/java]: Ah yeah I guess so :)

File: bin/java (hunk 1 above)
Comment (hone) [reply bin/java]: I believe the version string is 9+181 but the tarball on s3 are 9-181

Comment (jkutner) [reply bin/java]: Yes, the tarball can't have the `+` char in it. When you try to access it as an HTTP URL it 404s.

**Pull request merged**